docs/DafnyRef/UserGuide.md

```diff
@@ -101,36 +101,63 @@ on the [Dafny INSTALL page in the wiki](https://github.com/dafny-lang/dafny/wiki
 
 `dafny` is a conventional command-line tool, operating just like other
 command-line tools in Windows and Unix-like systems.
-
-
-- The format of a command-line is determined by the shell program that is executing the command-line (.e.g. bash, the windows shell, COMMAND, etc.). The command-line typically consists of file names and options, in any order, separated by spaces.
+In general, the format of a command-line is determined by the shell program that is executing the command-line (.e.g. bash, the windows shell, COMMAND, etc.), but is expected to be a series of space-separated "words", each representing a command, option, or file. 
+
+As of v3.9.0, `dafny` uses a subcommand-style command-line (like `git` for example); prior to v3.9.0, the 
+command line consisted only of options and files.
+It is expected that additional commands will be added in the future.
+Each command may have its own subcommands and its own options, in addition to generally applicable options. 
+Thus the format of the command-line is
+a command name, followed by 0 or more subcommands, followed by options and files:
+`dafny <command> <subcommand>* <options> <files>`
+
+The command-line `dafny <command> --help` gives help information for that particular <command>.
+
+Also, the command-style command-line has modernized the syntax of options. Like many other tools, options
+now typically begin with a double hyphen, with some options having a single-hyphen short form, such as `--help` and `-h`.
+The value of an option is given after an `=` character, rather than a `:`, as in `--target=java`
```

Comment:
You can use a space, `:` or `=` as the delimiter between an option and its argument.